### ISSUE TITLE
Import existing snapshots into MongoDB

### ISSUE THREAD

**Issue by MattiSG** [ambanum/OpenTermsArchive#732]
Following #714, import the existing https://github.com/ambanum/OpenTermsArchive-snapshots repository in the newly created MongoDB database, and then:

- [x] Document testing protocol
- [x] Provide a clear (not necessarily clean) script for import
- [x] Import
- [x] Test
- [x] Import historical snapshots (from before December 2021) in the production database
- [x] Measure performance gains by comparing the duration of a refilter on the full database in the Git version and the MongoDB version
- [x] Report publicly on the results in the decision record 0003.

**Comment by Ndpnt** [comment]
# Testing protocol

- Check if we can find entries in MongoDB database with almost empty content (less than 1000 characters) with:
```
db.snapshots.find({ 
    "content": { "$exists": true },
    "$expr": { "$lt": [ { "$strLenCP": "$content" }, 1000 ] },
    "mimeType": "text/html; charset=utf-8"
})
```
> Result: none was found

- Check if the OpenTermsArchive-snapshots number of commits and the number of entries in the database are of the same order of magnitude. It can't be exactly the same as the servers was not exactly started at the same time.

For git:
```
git rev-list --count HEAD --since="Dec 14 2021" --before="Jan 1 2022"
```

For mongo
```
db.snapshots.find({fetchDate:{$gte:ISODate("2021-12-14"), $lt:ISODate("2022-01-01")}}).count()
```
> Result: 26705 commits vs 26230 entries in the database

**Comment by Ndpnt** [comment]
Fixed by #739.
